lib/content-services/src/lib/content-node-selector/content-node-selector.service.spec.ts

```diff
@@ -1,120 +0,0 @@
-/*!
- * @license
- * Copyright 2019 Alfresco Software, Ltd.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { TestBed } from '@angular/core/testing';
-import { setupTestBed } from '@alfresco/adf-core';
-import { ContentNodeSelectorService } from './content-node-selector.service';
-import { ContentTestingModule } from '../testing/content.testing.module';
-import { TranslateModule } from '@ngx-translate/core';
-
-describe('ContentNodeSelectorService', () => {
```

Comment:
can we move those tests in the component to be sure they return the same result instead to only delete it?

Reply:
I moved them here https://github.com/Alfresco/alfresco-ng2-components/pull/6214/files#diff-27736d3aee3f7dd1d6a659f47375c8db52f3c4b50be05e4b39a1836b8963e856R622 . Not all of them were needed as some of them are already covered by the search-query-builder service